tests/network/l2_bridge/conftest.py

```diff
@@ -391,11 +393,14 @@ def configured_l2_bridge_vm_a(l2_bridge_running_vm_a):
 @pytest.fixture()
 def started_vmb_dhcp_client(l2_bridge_running_vm_b, eth3_nmcli_connection_uuid):
     nmcli_cmd = "sudo nmcli connection"
-    # Start dhcp client in l2_bridge_running_vm_b
+    # Use a unique DHCP client identifier to ensure only our test server responds
+
+    # Start dhcp client with unique client identifier
     run_ssh_commands(
         host=l2_bridge_running_vm_b.ssh_exec,
         commands=[
             shlex.split(f"{nmcli_cmd} modify '{eth3_nmcli_connection_uuid}' ipv4.method auto"),
+            shlex.split(f"{nmcli_cmd} modify '{eth3_nmcli_connection_uuid}' ipv4.dhcp-client-id '{UNIQUE_CLIENT_ID}'"),
```

Comment:
```suggestion
            shlex.split(f"{nmcli_cmd} modify '{eth3_nmcli_connection_uuid}' ipv4.method auto ipv4.dhcp-client-id '{UNIQUE_CLIENT_ID}'"),
```
Why not merge them into a single command?